schemas/app-sre/automated-action-argument-1.yml

```diff
@@ -13,30 +19,46 @@ properties:
     "$ref": "/common-1.json#/definitions/labels"
   type:
     type: string
-
-  # type=openshift related properties
+    description: |
+      The type of the automated action argument. This determines the 
+      specific configuration required for the argument.
   namespace:
     "$ref": "/common-1.json#/definitions/crossref"
     "$schemaRef": "/openshift/namespace-1.yml"
+    description: |
+      Reference to the OpenShift namespace associated with the argument. 
+      This is required when the type is `openshift`.
   kind_pattern:
     type: string
-    description: "The kind pattern to match."
+    description: |
+      The pattern used to match the kind of resources in OpenShift. 
+      This is required when the type is `openshift`.
   name_pattern:
     type: string
-    description: "The name pattern to match."
+    description: |
+      The pattern used to match the name of resources in OpenShift. 
+      This is required when the type is `openshift`.
 
 oneOf:
 - properties:
     type:
       enum:
       - openshift
+      description: |
+        Specifies that the argument is related to OpenShift resources.
```

Comment:
similar question, do we need to duplicate all description inside `oneOf`

Reply:
https://github.com/app-sre/qontract-schemas/pull/867#discussion_r2169554767 thoughts?